src/hooks/usePollDatasetStatus.js

```diff
@@ -47,8 +59,8 @@ export const usePollDatasetStatus = () => {
 
   const refreshProcessingDataset = async () => {
     const response = await getDataset(polledDatasetId)
-    // TEMP: until the fetchAsync is refactored
-    if (response?.ok !== false) {
```

Comment:
I've cleaned up the no longer required optional chaining and the strict equality check here to align with the latest `dev` (merged PR #337) from `refreshProcessingDataset`, thank you David.